firestore/google/cloud/firestore_v1beta1/client.py

```diff
@@ -582,8 +582,7 @@ def _parse_batch_get(get_doc_response, reference_map, client):
             a document factory.
 
     Returns:
-        Optional[.DocumentSnapshot]: The retrieved snapshot. If the
-        snapshot is :data:`None`, that means the document is ``missing``.
+        Optional[.DocumentSnapshot]: The retrieved snapshot.
```

Comment:
I don't know what `Optional` is supposed to mean for `Returns:`, but if it meant that the method might return `None`, that is no longer treu.